earth2studio/models/px/fcn3.py

```diff
@@ -278,6 +296,54 @@ def load_model(
 
         return cls(model, variables=variables)
 
+    def _get_internal_state(self, ensemble_index: int, time_index: int) -> torch.Tensor:
+        """Get the internal RNG state for the given ensemble and time index
+
+        Parameters
+        ----------
+        ensemble_index : int
+            Ensemble index
+        time_index : int
+            Time index
+        """
+        return self._internal_noise_states[ensemble_index][time_index]
```

Comment:
Does `self._internal_noise_states` need to be initialized in the constructor?

Like `self._internal_noise_states = []`

Its created when `_reset_internal_state` is called, is there a chance for this function to try to access this without it being initialized?

Maybe it starts as none and this can throw an initialization error that _reset_internal_state has not been called?